boilerplate/app/models/root-store/root-store-context.ts

```diff
@@ -0,0 +1,22 @@
+import { createContext, useContext } from "react"
+import { RootStore } from "./root-store"
+
+/**
+ * Create a context we can use to
+ * - Provide access to our stores from our root component
+ * - Consume stores in our screens (or other components, though it's
+ *   preferable to just connect screens)
+ */
+const RootStoreContext = createContext<RootStore>({} as RootStore)
```

Comment:
Interesting! I didn't know this generic was available.

Reply:
Yeah, discovered this when trying to get `useState` to type correctly - seems like all the hooks are typed to support generics! 🎉